src/d3a/models/area/redis_external_connection.py

```diff
@@ -0,0 +1,56 @@
+from redis import StrictRedis
+import json
+from d3a.d3a_core.redis_communication import REDIS_URL
+from d3a.models.strategy.external_strategy import ExternalStrategy
+
+
+class RedisExternalConnection:
+    def __init__(self, area):
```

Comment:
One more clarification, shouldn't there needs to have an implementation to unsubscribe external connection like we have for [RedisMarketCommunicator](https://github.com/gridsingularity/d3a/pull/567/files#diff-326ea26bc50c7cb2ae739e6ac2e2774bR65) ?

Reply:
That's a good question. In reality, we probably need an API endpoint that unregisters the user from the simulation (even though we do not need it immediately, I am almost sure that this is going to be a requirement for the future). I will try to expose another channel on the area redis subscribe class, so that a user can not only register to a simulation, but unregister from that too. Thanks!

Reply:
On second thought, its not an immediate requirement, so better create a separate enhancement story else this PR may unnecessary be delay to be resolved :-)